### PR TITLE
set status bar color on contributors screen

### LINKED ISSUE #212: [Assigned] StatusBar color on Contributor screen is different from other screens
## Overview (Required)
StatusBar color on Contributer screen is not primary dark color. This is defferent from other screens.  
I'm confirming this phenomenon with the Pixel2 emulator(API 24) and Nexus 5X(API 27).

## Screenshot
|Contributor screen|Session detail screen|
|---|---|
|||  

These screenshots were taken with Nexus 5X.

### PR DESCRIPTION
**Pull request by KensukeIzumi**
## Issue
- close #212

## Overview (Required)
- add drawer layout for setting status bar color

## Links
-

## Screenshot
Before | After
:--: | :--:
screenshot_20180116-140053 | screenshot_20180116-135920

Before | After
:--: | :--:
screenshot_20180116-140053 | screenshot_20180116-172440

### CODE REVIEW

**Comment by takahirom** [comment]
👀

**Comment by takahirom** [comment]
Would you add NavigationView and that's setup? 🙇 
https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/main/res/layout/activity_about_this_app.xml#L43

**Comment by ogapants** [comment]
@takahirom 
It was deleted by @konifar 's review...🤔 Are you OK?
https://github.com/DroidKaigi/conference-app-2018/pull/206#discussion_r161580120

**Comment by takahirom** [comment]
Sorry. Please wait a munute.🙇

**Comment by takahirom** [comment]
Please add NavigationView 👍
I talk with konifar.
Currently, We added to all activity. 
Because It is not good that only the contributor screen has no navigation drawer

**Comment by ogapants** [comment]
I agree with that opinion 😃

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/ContributorsActivity.kt [hunk 1]
```diff
@@ -35,6 +35,7 @@ class ContributorsActivity : BaseActivity(), HasSupportFragmentInjector {
         setSupportActionBar(binding.toolbar)
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
         navigationController.navigateToContributor()
+        drawerMenu.setup(binding.toolbar, binding.drawerLayout, binding.drawer)
     }
```
Comment (ogapants) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/ContributorsActivity.kt]: Please add `onBackPressed()` for closing drawer :bow:
```
override fun onBackPressed() {
     if (drawerMenu.closeDrawerIfNeeded()) {
         super.onBackPressed()
     }
}
```
ref: https://github.com/DroidKaigi/conference-app-2018/pull/145

Comment (KensukeIzumi) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/ContributorsActivity.kt]: Thanks for pointing out. I added that one!

Comment (ogapants) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/contributor/ContributorsActivity.kt]: @KensukeIzumi Great! Thanks! (Excuse me for cutting in...)

**Comment by takahirom** [comment]
LGTM! 💯

**Comment by takahirom** [comment]
@ogapants Thank you for a review! Thanks to you we were helped.

**Pull request merged**